### PR TITLE
Fix for issue #18

### PR DESCRIPTION
**Pull request by MitchBomcanhao**

### CODE REVIEW

**Comment by MitchBomcanhao** [comment]
This will probably also fix #17

File: TouchEffect.UWP/PlatformTouchEff.cs [hunk 1]
```diff
@@ -44,21 +46,31 @@ protected override void OnDetached()
...
+            if (_pressed)
+            {
+                _pressed = false;
+                Element.GetTouchEff().HandleTouch(TouchStatus.Completed);
+            }            
         }
 
         private void OnPointerPressed(object sender, PointerRoutedEventArgs e)
         {
+            this._pressed = true;
```
Comment (AndreiMisiukevich) [inline TouchEffect.UWP/PlatformTouchEff.cs]: please remove "this" here

**Review by AndreiMisiukevich — CHANGES_REQUESTED** [review]

**Comment by AndreiMisiukevich** [comment]
@MitchBomcanhao oh.. you make PR from master..
it would be better to create branch and make changes in separate branch

**Comment by AndreiMisiukevich** [comment]
@MitchBomcanhao as you wish. Please resolve small comment :) then ok

File: TouchEffect.UWP/PlatformTouchEff.cs (hunk 1 above)
Comment (AndreiMisiukevich) [inline TouchEffect.UWP/PlatformTouchEff.cs]:
```suggestion
            _pressed = true;
```

File: TouchEffect.UWP/PlatformTouchEff.cs [hunk 2]
```diff
@@ -21,6 +21,8 @@ public static void Preserve()
 
         private TouchEff _effect;
 
+        private bool _pressed = false;
```
Comment (AndreiMisiukevich) [inline TouchEffect.UWP/PlatformTouchEff.cs]:
```suggestion
        private bool _pressed;
```

**Review by AndreiMisiukevich — CHANGES_REQUESTED** [review]

**Comment by MitchBomcanhao** [comment]
Is something still missing? 
This page still says there is a change request, but I think I've approved everything I could.

**Review by AndreiMisiukevich — APPROVED** [review]

**Pull request merged**